### PR TITLE
Added camera pose for sensor to be in default view

### PR DESCRIPTION
**Pull request by quarkytale**
This is to slightly improve the DVL Seabed Gradient Demo by adding a GUI camera pose to include the teledyne sensor in the initial view.
Here's the GUI default view before and after the changes using the dave_dvl_gradient_demo_uuvsim.launch script:
Before
uuv default

After
uuv new

**To test**
Run either launch scripts to start gazebo and load a standalone DVL model containing a ocean world with a seabed and teledyne sensor. 

```
roslaunch dave_demo_launch dave_dvl_gradient_demo_uuvsim.launch
roslaunch dave_demo_launch dave_dvl_gradient_demo_dsl.launch
```
It should have the same effect with both launch scripts.

@mabelzhang would you please take a look!

### CODE REVIEW

File: models/dave_worlds/worlds/dave_graded_seabed.world
Comment (mabelzhang) [inline models/dave_worlds/worlds/dave_graded_seabed.world:85]: So, I'm having second thoughts (of course I didn't try this task myself before asking you to do it as a first practice PR). I'd like a second opinion from @dtdavi1.

@dtdavi1 For context, this PR (that Dharini is practicing with as her first PR) is adding a default camera angle as a small user experience improvement, because we realized that when a new user launches the world, they see the seabed, but they don't see the sensor as described in the tutorial. So they might (and did) go through troubleshooting to "fix" why their sensor isn't showing up. We thought adding a default camera angle closer to the camera would help.

Now taking a look at how things are loaded, the camera pose numbers here are inevitably very specific, while the sensor is loaded in a separate launch file (`upload_teledyne_whn.launch`), not in this world file. So hardcoding the numbers in this file is not the best style. While this world file is currently only used by the two DVL launch files, there's nothing preventing one from writing a launch file in the future and using this world file for something completely different from the DVL, which would make these camera angles obsolete for that use case. 

I might even say pass it in from the top-level launch file, but that would unnecessarily complicate things.

Maybe a simpler fix is a "tutorial improvement" as opposed to "code improvement"?

**Review by mabelzhang — COMMENTED** [review]
Looks good! I can reproduce the new default view.

Could you add screenshots for before and after, so that others know what the effect of this PR is, since it is a GUI improvement?

I have some minor comments and theoretical second thoughts (the latter is independent of what you've done)...

File: models/dave_worlds/worlds/dave_graded_seabed.world
Comment (quarkytale) [reply models/dave_worlds/worlds/dave_graded_seabed.world:87]: My GUI was coming out darker than in the tutorial wiki, still trying to find out why, but the change is evident.

Comment (dtdavi1) [reply models/dave_worlds/worlds/dave_graded_seabed.world:87]: I have no idea why, but all of the worlds come out darker on some computers.  There are some additional light sources in the world files that you can uncomment if desired (if they were uncommented by default, the scenes would be washed out on computers that they look good on now).

File: models/dave_worlds/worlds/dave_graded_seabed.world
Comment (dtdavi1) [reply models/dave_worlds/worlds/dave_graded_seabed.world:85]: I don't have strong feelings about it.  The camera isn't positioned well for some of the demos (it's not even close in the DVL demos) that reuse world files from other demos.  If the world in question isn't being used by any other demos anymore, it might be easiest just to move the camera in the world file.

File: models/dave_worlds/worlds/dave_graded_seabed.world
Comment (mabelzhang) [reply models/dave_worlds/worlds/dave_graded_seabed.world:87]: Note from @scpeters: see if there is error in `/home/$USER/.gazebo/ogre.log`

Comment (mabelzhang) [reply models/dave_worlds/worlds/dave_graded_seabed.world:87]: Note from William: possibly due to different driver versions. NVIDIA driver 460 in installation tutorial vs 510 that @quarkytale has. Maybe not a problem, more likely an issue if driver very out of date.

File: models/dave_worlds/worlds/dave_graded_seabed.world
Comment (mabelzhang) [reply models/dave_worlds/worlds/dave_graded_seabed.world:85]: Thanks for the input! Okay, guess we'll keep this here.

I would suggest though @quarkytale that a comment be added right above the `<pose>` line, that this pose is meant to be for the DVL demos - and name the two launch files. That way, if in the future someone does decide to reuse this world for another launch file, they'll see this and refactor the camera instead of changing it and breaking the DVL demos.

File: models/dave_worlds/worlds/dave_graded_seabed.world
Comment (mabelzhang) [reply models/dave_worlds/worlds/dave_graded_seabed.world:87]: If this turns out non-trivial and requires a longer thread of discussions, we should decouple it from this PR, so that we can unblock and move forward with this PR and continue that discussion in a separate ticket.

In that case, I would suggest @quarkytale opens an issue ticket documenting this, linking to the DVL tutorial that shows the lighter world, and this ticket showing the darker world, and to tag @WilliamLewww in that ticket to continue the conversation.

File: models/dave_worlds/worlds/dave_graded_seabed.world
Comment (quarkytale) [reply models/dave_worlds/worlds/dave_graded_seabed.world]: Updated as per your suggestion @mabelzhang!

**Comment by mabelzhang** [comment]
Looks good. Approval pending green CI.

Could you open an issue per https://github.com/Field-Robotics-Lab/dave/pull/186#discussion_r799931253 , to keep track of the light/dark rendering issue? I think we have some screenshots floating around showing both cases. It'd be good to include a comparison in the issue.

**Review by mabelzhang — APPROVED** [review]

**Pull request merged**
